e2e/davinci-suites/src/basic.test.ts

```diff
@@ -26,3 +26,45 @@ test('Test happy paths on test page', async ({ page }) => {
   const accessToken = await page.locator('#accessTokenValue').innerText();
   await expect(accessToken).toBeTruthy();
 });
+test('ensure query params passed to start are sent off in authorize call', async ({ page }) => {
```

Comment:
copy pasted above test, passed in a query param and then also added a few other query params from authorize specifically to make sure thats what we are testing